src/components/views/HomePage.js

```diff
@@ -2,9 +2,76 @@ import React from 'react';
 import {useHistory} from 'react-router-dom';
 import BaseContainer from "components/ui/BaseContainer";
 import 'styles/views/HomePage.scss';
+import { useEffect } from 'react';
+import Lobby from 'models/Lobby';
```

Comment:
I think should be: 

import Lobby from "./Lobby";

I get an error with "models/Lobby"

Reply:
thanks 👍